src/store/reducer.js

```diff
@@ -0,0 +1,16 @@
+const initialState =[];
+export default (state = initialState, action)=> {
+  let {type, payload={} } = action;
+  console.log('r-4',payload.status)
+  switch (type) {
+    case "GET":
+    return [...state,payload.data];
+
+    case"POST":
+    return [...state, payload.data];
+    case"DELETE":
+    return [...state, payload.status];
```

Comment:
Not sure why this has `payload.status` while the others have `payload.data`. More generally, I'm curious why the switch to Redux led you to preserve a list of responses. Previously you were just replacing `state.data` with the new response. A simpler reducer might look like:

```
switch (type) {
  case "SET_RESPONSE":
    return payload;
  default:
    return state;
}
```

I'll also just reinforce my comment above: if individual methods don't need to be handled differently, then you don't need to dispatch them as unique action types.